tools/theme/src/Theme.ts

```diff
@@ -158,38 +175,58 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
 
     public override shadowRoot!: ShadowRootWithAdoptedStyleSheets;
 
-    private _theme: ThemeVariant | '' = 'spectrum';
-
+    private _system: SystemVariant | '' = 'spectrum';
     /**
-     * The Spectrum theme that is applied to the content scoped to this `sp-theme` element.
+     * The Spectrum system that is applied to the content scoped to this `sp-theme` element.
      *
      * A value is requried.
-     * @type {"spectrum" | "express" | ""}
+     * @type {"spectrum" | "express" }
      * @attr
      */
-    get theme(): ThemeVariant | '' {
-        const themeFragments = Theme.themeFragmentsByKind.get('theme');
+    get system(): SystemVariant | '' {
+        const systemFragments = Theme.themeFragmentsByKind.get('system');
         const { name } =
-            (themeFragments && themeFragments.get('default')) || {};
-        return this._theme || (name as ThemeVariant) || '';
+            (systemFragments && systemFragments.get('default')) || {};
+        return this._system || (name as SystemVariant) || '';
     }
 
-    set theme(newValue: ThemeVariant | '') {
-        if (newValue === this._theme) return;
-        const theme =
-            !!newValue && ThemeVariantValues.includes(newValue)
+    set system(newValue: SystemVariant | '') {
+        if (newValue === this._system) return;
+        const system =
+            !!newValue && SystemVariantValues.includes(newValue)
                 ? newValue
-                : this.theme;
-        if (theme !== this._theme) {
-            this._theme = theme;
+                : this.system;
+        if (system !== this._system) {
+            this._system = system;
             this.requestUpdate();
         }
-        if (theme) {
-            this.setAttribute('theme', theme);
+        if (system) {
+            this.setAttribute('system', system);
             /* c8 ignore next 3 */
         } else {
-            this.removeAttribute('theme');
+            this.removeAttribute('system');
+        }
+    }
+
+    /*
+     * @deprecated The `theme` attribute has been deprecated in favor of the `system` attribute.
+     */
+    get theme(): SystemVariant | '' {
+        if (!this.system) {
+            this.removeAttribute('system');
+        }
+        return this.system;
+    }
+
+    /*
+     * @deprecated The `theme` attribute has been deprecated in favor of the `system` attribute.
+     */
+    set theme(newValue: SystemVariant | '') {
+        if (this.hasAttribute('system')) {
+            // System is provided, so do not set theme
+            return;
```

Comment:
Being `theme` sets `system`, if a end use _only_ sets `theme`, they won't be able to set it a second time to a new value...

Reply:
Yes we should not return out if system attribute is found. 
```
set theme(newValue: SystemVariant | '') {
        this.system = newValue;
        this.requestUpdate();
    }
```